xarray/core/dataarray.py

```diff
@@ -4659,14 +4659,15 @@ def curvefit(
 
     def drop_duplicates(
         self,
-        dim: Hashable,
-        keep: (str | bool) = "first",
+        dim: Hashable | Iterable[Hashable] | ...,
+        keep: Literal["first", "last"] | Literal[False] = "first",
```

Comment:
It would be a breaking change to put this behind a kwarg-only separator, but probably we should do it at some point. Someone is going to have a dim called `last` and be confused why `da.drop_duplicates("foo", "last")` is not dropping things on the `last` dim...